README.md

```diff
@@ -24,41 +24,68 @@ The [BrainFlow](https://BrainFlow.org) library provides a uniform API that is de
 7. Turn on your headband
 8. Run the script `main.py` with your device id. The command for running with a [Muse 2 headband](https://choosemuse.com/muse-2/) would be: `python .\main.py --board-id 38`
 
-**OSC Avatar Parameters**
-
-Avatar parameters being sent are floats that range from -1.0 to 1.0. Negative and Positive values correspond to low and high focus/relaxation. Update your avatar paramaters as needed to influnece animations and the like. Parameters are also seperated by left and right sides of the brain. Have fun!
-
-- `osc_focus_left`
-- `osc_relax_left`
-- `osc_focus_right`
-- `osc_relax_right`
-- `osc_focus_avg`
-- `osc_relax_avg`
-
-For easier startup, I've added a hue shift parameter based on the focus and relax values. This parameter will range from 0.0 to 1.0.
-
-- `HueShift`
-
-Added are optional paramaters that appear based on whether or not your headband supports it.
-- `osc_battery_lvl` (int [0-100])
-- `osc_heart_bpm` (int[0-255])
-- `osc_oxygen_percent` (float[0.0-1.0])
-- `osc_respiration_bpm` (int[0-255])
-
-I've also added the alpha, beta, theta, delta, and gamma band power ratios between left, right sides of the head as well as avg. You can access them via this path:
-- `osc_band_power_(head_location)_(brainwave name)` (float [0-1]) (without the parenthesis)
-
-For debug, a boolean paramater is sent to monitor the connection status of the headband as well as a time difference parameter to show the difference between last sample time and current time in seconds.
-
-- `osc_is_connected` (boolean)
-- `osc_time_diff` (float)
+**OSC Avatar Parameter Schema**
+
+Here are the various avatar parameters sent to VRChat. Focus and relax scores range from -1 to 1, corresponding to much focus and relaxation you're experiencing. Brain wave power bands are also sent as floats from 0 to 1. Depending on the board you're using, respiration data might be available.
+
+```yaml
+Brainflow:

```

Comment:
Additional proposal; I'm wondering if it's worthwhile to include a `Meta` scope to the base list (and perhaps another one in the device list at a later point). This could include things like schema version numbers which could be used to determine compatibility with current prefabs.

```yaml
Brainflow:
  Meta:
    - VersionMajor [int] -- Determines breaking changes in the schema representation
    - VersionMinor [int] -- Any update to the schema which remains compatible with existing prefabs. Useful to ensure that a feature a prefab requires is supported by the current application
```

Unlike other parts of the schema, this would not require data from any devices, and would be managed solely by the application itself. The major and minor fields should report `1` and `0` respectively to indicate the first release version of the schema.